src/chrome/content/rules/ooo.0o0.ooo.xml

```diff
@@ -0,0 +1,7 @@
+<ruleset name="ooo.0o0.ooo">
+	<target host="ooo.0o0.ooo" />
+
+	<securecookie host=".+" name=".+" />
```

Comment:
Does this domain set any cookies?

Reply:
I'm not sure, but if it doesn't, `securecookie` will simply do nothing.